client/packages/system/src/Store/components/StoreSearchInput/StoreSearchInput.tsx

```diff
@@ -20,6 +21,14 @@ type StoreSearchInputProps = {
   ) => void;
 };
 
+const filterByNameAndCode = (options: StoreRowFragment[], state: any) =>
```

Comment:
Instead of any can we put: FilterOptionsState<StoreRowFragment> ? Or can even add the filter inline in Autocomplete below